docs/WinML_principles.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to Windows ML
+
+Window Machine Learning is a high-performance, reliable API for deploying hardware-accelerated ML inferences on Windows devices. Please visit the [Windows ML documentation](https://docs.microsoft.com/en-us/windows/ai/windows-ml/) to learn more about Windows Machine Learning. 
```

Comment:
proposed change: "to learn more about Windows ML"

Reply:
fixed